doc/compiling/README.md

```diff
@@ -39,6 +39,8 @@ General options and their default values:
     ENABLE_UPDATE_CHECKER=TRUE - Whether to enable update checks by default
     INSTALL_DEVTEST=FALSE      - Whether the Development Test game should be installed alongside Minetest
     USE_GPROF=FALSE            - Enable profiling using GProf
+    BUILD_WITH_TRACY=FALSE     - Fetch and build with the Tracy profiler client
+    FETCH_TRACY_GIT_TAG=master - Git tag for fetching Tracy client. Match with your server (gui) version
```

Comment:
`TRACY_FETCH_GIT_TAG` or `TRACY_GIT_TAG` would fit better into the existing name scheme of libraries.

Also: `master` is not a tag. Either this example is incorrect or the variable name is misleading.

Reply:
I chose not to use a `TRACY_` prefix, because all of tracy's cmake options start with `TRACY_`, e.g. `TRACY_ENABLE`. Like this, it's easier to see which variables come from us.

The git tag is for FetchContent's `GIT_TAG`, accepts branch names (and also commit hashes).
Do you have a suggestion for a better description?

Reply:
Perhaps `FETCH_TRACY_GIT_REV`? 

https://git-scm.com/docs/gitrevisions

Reply:
Oops, forgot this here.

Revisions also allow commit hashes. But because I've set `GIT_SHALLOW` to true, `GIT_TAG` doesn't support commit hashes (according to doc at least).
I'd therefore prefer to just keep it like this.